packages/react-components/react-nav-preview/src/components/useNavContextValues.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { NavContextValue, NavContextValues, NavState } from '../Nav';
+
+export function useNavContextValues_unstable(state: NavState): NavContextValues {
+  const { selectedValue: selectedKey, onRegister, onUnregister, onSelect, getRegisteredNavGroups } = state;
+
+  const navContext = React.useMemo<NavContextValue>(
+    () => ({
+      selectedValue: selectedKey,
+      onSelect,
+      onRegister,
+      onUnregister,
+      getRegisteredNavGroups,
+    }),
+    [selectedKey, onSelect, onRegister, onUnregister, getRegisteredNavGroups],
+  );
```

Comment:
super-nit: What's the reason you're locally renaming `selectedValue` to `selectedKey`?

Reply:
Copy pasta, I think. 🍝

Reply:
✅